readme.md

```diff
@@ -57,6 +65,22 @@ function handle(stream) {
 }
 ```
 
+We recommend disabling the [per message deflate
+extension](https://tools.ietf.org/html/rfc7692) to achieve the best
+throughput:
+
+```javascript
+var websocket = require('websocket-stream')
+var wss = websocket.createServer({
+  perMessageDeflate: false,
+  server: someHTTPServer
+}, handle)
+
+function handle(stream) {
+  fs.createReadStream('bigdata.json').pipe(stream)
+}
+```
+
```

Comment:
It might makes sense to show how to disable it on the client (it's the same) as `websocket-stream` could be used only as a client.